package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "sphinx_rtd_theme",
   "main": "js/theme.js",
-  "version": "1.0.1alpha1",
+  "version": "1.1.0-alpha.0",
```

Comment:
Bumpversion and bumpver will put preference on the existing versioning pattern, I believe, so this will likely be overwritten. Bumpversion and bumpver follow PEP 440, which does not use the optional period in between:

`1.0.1alpha1`
`1.0.1alpha2`

https://peps.python.org/pep-0440/#pre-releases

I'd say lets stick with the pattern described in PEP440, both are equally valid as far as I know.

Reply:
Also, setuptools normalizes this automatically anyways:

```
/home/anthony/secure/dev/readthedocs/theme/.direnv/python-3.8.12/lib/python3.8/site-packages/setuptools/dist.py:463: UserWarning: Normalizing '1.0.1alpha1' to '1.0.1a1'
  warnings.warn(tmpl.format(**locals()))
```

Reply:
Also note, changing this string makes bumpversion not work, unfortunately, as it won't try to change this string anymore

Reply:
We'll resolve this in #1326